src/dmd/libmach.d

```diff
@@ -213,7 +218,10 @@ final class LibMach : Library
         om.scan = 1;
         if (fromfile)
         {
-            stat_t statbuf;
+            version (Posix)
+                stat_t statbuf;
+            version (Windows)
+                struct_stat statbuf;
```

Comment:
Using a renamed import at L27 might be more readable:
```
    import core.sys.windows.stat : stat_t = struct_stat;
```

Wouldn't it ?

Reply:
No, because the reviewer would see the `stat_t` later in the code and think: "hmm, Windows doesn't have a `stat_t`, why is this compiling?"

Things that try to fake one operating system to look like another are ultimately not worth it. I've done a lot of this stuff in the past, and it just makes things worse.

Reply:
> why is this compiling ?

Then ctrl + F or `grep` for `stat_t` and get the answer. Or scroll up to the imports.

> Things that try to fake one operating system to look like another are ultimately not worth it.

Well it's the same approach, just a different name for the type of the parameter :/

Reply:
As I said, been there, done that, got the kewpie doll, too many times :-)